hub/developer_models_compact_index.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: hub_index
+title: For Developers
+permalink: hub/developer-models/compact
+background-class: hub-background
+body-class: hub
+summary: Get plug & play models to accelerate ML development.
+compact: true
+category: developer
```

Comment:
The category should be "developers". Sorry, I think I threw you off by adding it incorrectly to `developer_models_index.html`. Either way, I think you can remove the category variable from all of the hub index pages since we're not calling `page.category` anywhere. (It gets called on `hub_detail.html`, but it comes from the hub markdown file.)